scripts/estimate-observed-transmission-parameters.R

```diff
@@ -71,4 +71,4 @@ fwrite(
 fwrite(
   posterior_predictions,
   here::here("data", "retrospective", "posterior_predictions.csv")
```

Comment:
So this overwrites the previous estimates? it might be better to save under a different name or duplicate the script with changes?